azure/table_azure_firewall.go

```diff
@@ -261,6 +261,8 @@ func ipConfigurationData(ctx context.Context, d *transform.TransformData) (inter
 	data := d.HydrateItem.(network.AzureFirewall)
 
 	var output []map[string]interface{}
+// Add a check for AzureFirewallPropertiesFormat.IPConfigurations data to ensure that
+// it is not null to avoid panic errors
```

Comment:
```suggestion
	// Add a check for AzureFirewallPropertiesFormat.IPConfigurations data to ensure that
	// it is not null to avoid panic errors
```

@mdaguete Can you please fix the spacing a bit? Thanks!